plugins/search-react/src/components/SearchBar/SearchBar.tsx

```diff
@@ -147,14 +141,22 @@ export const SearchBarBase: ForwardRefExoticComponent<SearchBarBaseProps> =
 
       return (
         <TrackSearch>
-          <InputBase
+          <TextField
+            id="search-bar-text-field"
             data-testid="search-bar-next"
-            ref={ref}
+            variant="outlined"
+            margin="normal"
+            inputRef={ref}
             value={value}
-            placeholder={placeholder}
-            startAdornment={startAdornment}
-            endAdornment={clearButton ? endAdornment : defaultEndAdornment}
-            inputProps={{ 'aria-label': 'Search', ...defaultInputProps }}
+            label={label}
```

Comment:
I guess we have a couple of storybook examples that should be updated with this change as well? For example https://backstage.io/storybook/?path=/story/plugins-search-searchbar--custom-placeholder

Reply:
Have you tested it in localhost? I would expect the changes in the component would reflect in the stories, will check

Reply:
Sorry I got now what you meant, instead of custom placeholder, it should be custom label now, will fix in the story

Reply:
Done, added a new story just for label & updated the paddings in stories.